testkit/tests/counter/main.rs

```diff
@@ -417,141 +416,215 @@ fn test_snapshot_comparison_panic() {
         });
 }
 
+fn create_sample_block(testkit: &mut TestKit) {
+    let height = testkit.height().next().0;
+    if height == 2 || height == 5 {
```

Comment:
I think explanation why `2` and `5` are used will be helpful for those who'll read this in the future :)